docs/docs/upgrading.md

```diff
@@ -7,6 +7,28 @@ description: >-
 
 # Since 0.15.0
 
+## New
+
+### Policy for Device Identity
+
+This release of Pomerium adds the ability to set policy based on system registration via [WebAuthN](https://www.w3.org/TR/webauthn-2/#registration-extension).
```

Comment:
@travisgroth I don't understand the reasoning to link to that specific header.

Reply:
IMHO, we should link to the wikipedia entry. https://en.wikipedia.org/wiki/WebAuthn